datahub-web-react/src/app/identity/user/UserListItem.tsx

```diff
@@ -45,12 +66,17 @@ const MenuIcon = styled(MoreOutlined)<{ fontSize?: number }>`
     margin-left: 5px;
 `;
 
-export default function UserListItem({ user, canManageUserCredentials, onDelete }: Props) {
+export default function UserListItem({ user, canManageUserCredentials, roles, onDelete, refetch }: Props) {
     const entityRegistry = useEntityRegistry();
     const [isViewingResetToken, setIsViewingResetToken] = useState(false);
+    const [isViewingAssignRole, setIsViewingAssignRole] = useState(false);
```

Comment:
Wait wait wait --- it looks like we are overloading the UserListItem here to display ROLES?
 
We should have a separate RolesList and RolesListItem component completely in their own hierarchy. It's too messy to maintain these 2 things as merged

Reply:
I'm not sure how we can have a RolesList since it really is just a UserList, but with this select component on the UserListItem. Maybe we can just refactor that part?

Reply:
yeah i think so

Reply:
okay cool marking this as resolved since we have the select now